src/main/java/com/endava/endabank/dto/user/AuthenticationDto.java

```diff
@@ -1,10 +1,26 @@
 package com.endava.endabank.dto.user;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
 
 @Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
 public class AuthenticationDto {
+    @Email
+    @NotNull
     private String email;
+
+    @NotNull
     private String password;
 
+    public void setEmail(String email){
+        this.email = email.toLowerCase();
```

Comment:
Hi,

When you have a dto class is not a good practice to have logic like this toLowerCase(), it would be better if you have that part in another utility class or when you need it.

http://www.juntadeandalucia.es/servicios/madeja/sites/default/files/historico/1.3.0/contenido-libro-pautas-38.html

"Un DTO normalmente no provee lógica de negocios o validaciones de ningún tipo. Solo provee acceso a las propiedades del bean."